homeassistant/components/august/activity.py

```diff
@@ -140,11 +145,25 @@ def async_schedule_house_id_refresh(self, house_id: str) -> None:
 
         debouncer = self._update_debounce[house_id]
         debouncer.async_schedule_call()
+
         # Schedule two updates past the debounce time
         # to ensure we catch the case where the activity
         # api does not update right away and we need to poll
         # it again. Sometimes the lock operator or a doorbell
         # will not show up in the activity stream right away.
+        # Only do additional polls if we are past
+        # the initial lock resync time so avoid a storm
+        # of activity at setup.
+        if (
+            not self._start_time
+            or monotonic() - self._start_time < INITIAL_LOCK_RESYNC_TIME
+        ):
+            _LOGGER.debug(
+                "Skipping additional updates because we are in the initial lock resync time"
```

Comment:
optional, but might be nice to avoid "we" in a logger message:
```suggestion
                "Skipping additional updates due to ongoing initial lock resync time"
```

Reply:
thanks